### DISCUSSION TITLE
An inquiry about the total Lagrangian formulation of ComputeMultipleCrystalPlasticityStress class

### DISCUSSION THREAD

**Discussion by NINGYURICHARD** [idaholab/moose#28248] [Q&A Modules: Solid mechanics] [answered]
Hello everyone!

Could anyone tell me whether the ComputeMultipleCrystalPlasticityStress class contains the total Lagrangian formulation? I noticed that the updated Lagrangian incremental form is used in this class on 

https://mooseframework.inl.gov/source/materials/crystal_plasticity/ComputeMultipleCrystalPlasticityStress.html

Thank you very much!

Regards,
Richard

**Comment by GiudGiud** [comment]
@dewenyushu

**Comment by dewenyushu** [comment, marked as answer]
No, we only have this form in this class. As far as I know, this is the only form available in MOOSE.

**Reply by NINGYURICHARD** [reply]
Thank you!
